include/arch/x86/acpi.h

```diff
@@ -8,19 +8,17 @@
 
 #ifndef _ASMLANGUAGE
 
-extern void *z_acpi_find_table(uint32_t signature);
-
 /* Standard table header */
 struct acpi_sdt {
-	uint32_t sig;
-	uint32_t len;
-	uint8_t rev;
-	uint8_t csum;
-	char oemid[6];
+	uint32_t signature;
+	uint32_t length;
+	uint8_t revision;
+	uint8_t chksum;
+	char oem_id[6];
```

Comment:
No complaints about longer symbol names obviously.  Let's just say though that I'm amused about the irony here in ACPI code, an environment that baked a 4-digit symbol name limit into its specification.